xbmc/pvr/guilib/guiinfo/PVRGUIInfo.cpp

```diff
@@ -307,12 +307,15 @@ namespace
     {
       if (CServiceBroker::GetPVRManager().IsParentalLocked(epgTag))
         return g_localizeStrings.Get(19266); // Parental locked
-      else if (!epgTag->Title().empty() ||
-               CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
-                   CSettings::SETTING_EPG_HIDENOINFOAVAILABLE))
+      else if (!epgTag->Title().empty())
```

Comment:
This looks wrong. Now, if there is a non-empty epg tag title, the "hide" setting is ignored.

Reply:
Could it be that the original `|| CService` simply needs to be changed to `&& !CService...`?

Reply:
If it’s not empty there is nothing to hide.

Reply:
OMG. You are ofc right.